dist/ng-inline-edit.js

```diff
@@ -254,6 +256,12 @@
             attrs.$observe('inlineEditPlaceholder', function(placeholder) {
               scope.placeholder = placeholder;
             });
+
+            scope.$watch("model", function(newValue, oldValue) {
+              if (!isNaN(parseFloat(newValue)) && isFinite(newValue) && newValue === 0) {
```

Comment:
@tameraydin you are totally right. The number check is not needed anymore here. ```newValue === 0``` suffices.